replication_test.go

```diff
@@ -1085,3 +1087,331 @@ func TestReplicationVerifyEmptyNotarization(t *testing.T) {
 		return wal.ContainsEmptyNotarization(0)
 	}, time.Millisecond*500, time.Millisecond*10, "Did not expect an empty notarization with a corrupt QC to be written to the WAL")
 }
+
+// almostFinalizeBlocks is a message filter that allows all messages except for finalized votes
```

Comment:
Can we move this method to the test it's used in, or alternatively, parametrize `simplex.NodeID{1}` ? It seems highly specific to the test it's used at, and therefore it should really be defined in the test.